.devcontainer/compose.yaml

```diff
@@ -6,30 +6,12 @@ services:
       context: ..
       dockerfile: .devcontainer/Dockerfile
     volumes:
+      - app-db:/workspaces/byos_sinatra/db/sqlite
       - ../..:/workspaces:cached
     environment:
-      DATABASE_URL: postgres://postgres:postgres@postgres:5432/postgres
+      APP_URL: https://localhost:4443
```

Comment:
:sparkle: Thanks. This was boilerplate so didn't get a chance to clean this up (especially since PostgreSQL and Redis isn't needed either).